mmdet/visualization/local_visualizer.py

```diff
@@ -333,6 +372,11 @@ def add_datasample(
         else:
```

Comment:
`drawn_img` passed to `self.set_image(drawn_img)` should not be None.

```python
elif pred_img_data is not None:
    drawn_img = pred_img_data
else:
    drawn_img = image
```

Reply:
drawn_img cannot be None